validmind/models/foundation.py

```diff
@@ -29,22 +29,16 @@ class FoundationModel(VMModel):
           and return the result from the model
         prompt (Prompt): The prompt object that defines the prompt template and the
           variables (if any)
-        attributes (ModelAttributes, optional): The attributes of the model. Defaults to None.
+        output_column (str, optional): The output column name where predictions are stored.
```

Comment:
> Defaults to the `input_id` plus `_prediction`

If that's the default behavior any reason to introduce a new `output_column` variable?